llvm/test/CodeGen/Hexagon/hwloop-dist-check.mir

```diff
@@ -0,0 +1,281 @@
+# RUN: llc -run-pass=hwloops %s -o - | FileCheck %s
```

Comment:
Needs `--mtriple=hexagon`.

This causes a CI failure.

Reply:
Thank you Brian. Fixed it